cql3/functions/user_aggregate.hh

```diff
@@ -10,23 +10,17 @@
 
 #include "scalar_function.hh"
 #include "cql3/description.hh"
-#include "data_dictionary/keyspace_element.hh"
 #include "cql3/functions/function_name.hh"
 #include "db/functions/aggregate_function.hh"
 
 namespace cql3 {
 namespace functions {
 
-class user_aggregate : public db::functions::aggregate_function, public data_dictionary::keyspace_element {
+class user_aggregate : public db::functions::aggregate_function {
```

Comment:
This is a good side effect, moving the responsibility for description outside where it can have more knowledge of the wider world.